hr_attendance_missing_days/models/hr_employee.py

```diff
@@ -79,6 +79,9 @@ def _create_missing_attendances(self, date_from=None, date_to=None):
         attendances = {
             ensure_tz(attendance.check_in, tz).date()
             for attendance in self.attendance_ids.filtered_domain(domain)
+        } | {
+            ensure_tz(attendance.check_out, tz).date()
+            for attendance in self.attendance_ids.filtered_domain(domain)
```

Comment:
I considered enumerating the days between check_in and check_out, but then I couldn't think of a real live case where anyone records 24+ hours

Reply:
Technically it's clean but the entire `self.attendance_ids.filtered_domain(domain)` would be executed multiple times. I guess the following could be better (well I'm not fully pleased with the naming)

```python
attendances = self.attendance_ids.filtered_domain(domain)
attendances = attendances.mapped("check_in") + attendances.mapped("check_out")
attendances = {ensure_tz(attendance, tz).date() for attendance in attendances}
```

Reply:
thanks, made it so